piecash/ledger.py

```diff
@@ -16,7 +16,7 @@ def _process(fct):
 
 
 @attach_ledger(Transaction)
-def ledger(tr):
+def ledger(tr, full_names=True):
```

Comment:
I wonder if it would not be better to use a **kwargs in each signature of the ledger function (to avoid having to change all signatures if we add other options like "full_names"), i.e.
`def ledger(tr, **kwargs):`
and for the one using the option have
`def ledger(tr, full_names=True, **kwargs):`

Would this make sense ?